memconfig.json

```diff
@@ -150,35 +154,63 @@
         "name": "Moderation",
         "fields": [
           {
-            "name": "violence/graphic",
```

Comment:
I also added a check so that invalid characters in field names are rejected by the API